server/org_config.go

```diff
@@ -0,0 +1,181 @@
+	Links selfLinks `json:"links"`
+	chronograf.OrganizationConfig
+}
+
+func newOrganizationConfigResponse(config chronograf.OrganizationConfig) *organizationConfigResponse {
+	return &organizationConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config",
+		},
+		OrganizationConfig: config,
+	}
+}
+
+type logViewerConfigResponse struct {
+	Links selfLinks `json:"links"`
+	chronograf.LogViewerConfig
+}
+
+func newLogViewerConfigResponse(lv chronograf.LogViewerConfig) *logViewerConfigResponse {
+	return &logViewerConfigResponse{
+		Links: selfLinks{
+			Self: "/chronograf/v1/org_config/logviewer",
+		},
+		LogViewerConfig: lv,
+	}
+}
+
+// OrganizationConfig retrieves the organization-wide config settings
+func (s *Service) OrganizationConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	orgID, ok := hasOrganizationContext(ctx)
+	if !ok {
+		Error(w, http.StatusBadRequest, "Organization not found on context", s.Logger)
+		return
+	}
+
+	config, err := s.Store.OrganizationConfig(ctx).FindOrCreate(ctx, orgID)
+	if err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	res := newOrganizationConfigResponse(*config)
+
+	encodeJSON(w, http.StatusOK, res, s.Logger)
+}
+
+// OrganizationLogViewerConfig retrieves the log viewer UI section of the organization config
+// This uses a FindOrCreate function to ensure that any new organizations have
+// default organization config values, without having to associate organization creation with
+// organization config creation.
+func (s *Service) OrganizationLogViewerConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	orgID, ok := hasOrganizationContext(ctx)
+	if !ok {
+		Error(w, http.StatusBadRequest, "Organization not found on context", s.Logger)
+		return
+	}
+
+	config, err := s.Store.OrganizationConfig(ctx).FindOrCreate(ctx, orgID)
+	if err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	if config == nil {
+		Error(w, http.StatusBadRequest, "Organization configuration object was nil", s.Logger)
+		return
+	}
+
+	res := newLogViewerConfigResponse(config.LogViewer)
+
+	encodeJSON(w, http.StatusOK, res, s.Logger)
+}
+
+// ReplaceOrganizationLogViewerConfig replaces the log viewer UI section of the organization config
+func (s *Service) ReplaceOrganizationLogViewerConfig(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	orgID, ok := hasOrganizationContext(ctx)
+	if !ok {
+		Error(w, http.StatusBadRequest, "Organization not found on context", s.Logger)
+		return
+	}
+
+	var logViewerConfig chronograf.LogViewerConfig
+	if err := json.NewDecoder(r.Body).Decode(&logViewerConfig); err != nil {
+		invalidJSON(w, s.Logger)
+		return
+	}
+	if err := validLogViewerConfig(logViewerConfig); err != nil {
+		Error(w, http.StatusBadRequest, err.Error(), s.Logger)
+		return
+	}
+
+	config, err := s.Store.OrganizationConfig(ctx).FindOrCreate(ctx, orgID)
```

Comment:
Having to do this here makes me think that i'd be good to just have an `UpdateLogViewerConfig` so that we don't need to do this lookup here.